src/cache/cache_las.c

```diff
@@ -86,16 +114,17 @@ __wt_las_create(WT_SESSION_IMPL *session)
 	WT_RET(ret);
 
 	/* Re-create the table. */
-	WT_RET(__wt_session_create(session, WT_LAS_URI, WT_LAS_FORMAT));
+	WT_RET(__wt_session_create(session, WT_LAS_URI, WT_LAS_CONFIG));
 
 	/*
 	 * Open a shared internal session and cursor used for the lookaside
-	 * table. This session should never be tapped for eviction.
+	 * table. This session should never perform reconciliation.
 	 */
-	session_flags = WT_SESSION_NO_EVICTION;
-	WT_RET(__wt_open_internal_session(
-	    conn, "lookaside table", true, session_flags, &conn->las_session));
-	WT_RET(__wt_las_cursor_open(conn->las_session));
+	for (i = 0; i < WT_LAS_NUM_SESSIONS; i++) {
+		WT_RET(__wt_open_internal_session(conn, "lookaside table",
+		    true, WT_LAS_SESSION_FLAGS, &cache->las_session[i]));
+		WT_RET(__wt_las_cursor_open(cache->las_session[i]));
+	}
```

Comment:
This now opens 5 sessions for lookaside access rather than the 1 we had before.

We could open these lazily, I think (the first time lookaside is required).  Worth doing?

Reply:
Do we need to think about adjusting `WT_EXTRA_INTERNAL_SESSIONS`?  That is currently 20, but with potentially many LSM threads, eviction threads, several logging threads, and others and now 5 of these, we could be encroaching on that.

Reply:
Good question - I've opened WT-3738 to address it.